src/utils/AccessibleModal.ts

```diff
@@ -13,6 +13,22 @@ export interface IAccessibleModalOptions {
   sizeMod: 'small' | 'normal' | 'big';
 }
 
+export interface IAccessibleModalOpenParameters {
+  content: HTMLElement;
+  validation: () => boolean;
+  origin?: HTMLElement;
```

Comment:
Why is the `origin` property optional? Couldn't we always specify it?

Reply:
I was hesitant about making it optional or mandatory and made it optional to reduce change, but I admit that I don't really see a purpose to making it optional.

I'll make it mandatory.